### PR TITLE
Reorganizes, adds new chaplain religions

### PR DESCRIPTION
**Pull request by Farquaar**
## About The Pull Request
This pull request alphabetizes the religion list (with the default religion at the top), and adds a number of new religions, including world religions, small religions and made up ones that I've seen in-game by roleplaying chaplains.

## Why It's Good For The Game
Variety is always nice, and the addition of new religions sets the groundwork for other ideas I have to flesh out in-game religions.

## Changelog
:cl:
add: Added several new religions to the Nanotrasen-approved faith list, including Judaism, Rastafarianism, and Honkism
/:cl:

### CODE REVIEW

**Comment by Militaires** [comment]
roll for anal circumference, also, add the church of subgenius.

**Comment by Farquaar** [comment]
Is that like Pastafarianism without the pasta?

**Comment by ATH1909** [comment]
Nice FATAL reference.

Maybe add Lampism as a religion, since it's a common moth chaplain religion?

**Comment by Bombpants** [comment]
What about boat Mormons?

**Comment by Militaires** [comment]
http://nightflight.com/wp-content/uploads/CHURCHOFSUBGENIUS4.jpg

**Comment by dr-bee** [comment]
Could I request you add "comedy" to the list of clown religions?

File: code/modules/jobs/job_types/chaplain.dm [hunk 1]
```diff
@@ -52,27 +52,49 @@
...
+		if("rastafarianism","rasta")
+			B.name = "The Holy Piby"
+		if("satanism")
+			B.name = "The Unholy Bible"
 		if("science")
 			B.name = pick("Principle of Relativity", "Quantum Enigma: Physics Encounters Consciousness", "Programming the Universe", "Quantum Physics and Theology", "String Theory for Dummies", "How To: Build Your Own Warp Drive", "The Mysteries of Bluespace", "Playing God: Collector's Edition")
+		if("scientology")
+			B.name = pick("The Biography of L. Ron Hubbard","Dianetics")
+		if("subgenius")
+			B.name = "Book of the SubGenius"
```
Comment (81Denton) [inline code/modules/jobs/job_types/chaplain.dm]: BASED and floorpilled

**Comment by 81Denton** [comment]
>F.A.T.A.L.
>Subgenius

Bless this PR

**Review by 81Denton — DISMISSED** [review]

File: code/modules/jobs/job_types/chaplain.dm [hunk 2]
```diff
@@ -52,27 +52,49 @@
...
+		if("toolboxia","greytide")
+			B.name = "Toolbox Manifesto"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/jobs/job_types/chaplain.dm]:
```suggestion
			B.name = pick("Toolbox Manifesto","iGlove Assistants")
```

**Comment by ExcessiveUseOfCobblestone** [comment]
Is this ok to merge

**Pull request merged**